unreleased.rst

```diff
@@ -27,3 +27,10 @@ DAML Integration Kit
 
 - Added additional Ledger API integration tests to Ledger API Test Tool.
 - [DAML Studio] Goto definition now works on the export list of modules.
+
+Navigator
+~~~~~~~~~~
+
+- Added support for SDK project configuration files. If you start Navigator with the SDK Assistant,
```

Comment:
The new format for unreleased release notes is to add the section heading inline to improve git merging, e.g.:

```suggestion
- [Navigator] Added support for SDK project configuration files. If you start Navigator with the SDK Assistant,
```